libs/message-queue/src/lib/nest/worker.service.ts

```diff
@@ -46,7 +47,19 @@ export class WorkerService implements OnModuleDestroy {
     )
 
     this.status = Status.Running
+
     while (this.status === Status.Running) {
+      const now = new Date()
```

Comment:
~Shouldn't need to pass now date to utils. Tests can mock the time with useFakeTimers.